src/discussions/common/ActionsDropdown.jsx

```diff
@@ -91,9 +91,18 @@ function ActionsDropdown({
                     close();
                     handleActions(action.action);
                   }}
-                  className="d-flex justify-content-start py-1.5 mr-4"
+                  className="d-flex justify-content-start actions-dropdown-item"
                 >
-                  <Icon src={action.icon} className="mr-1" /> {intl.formatMessage(action.label)}
+                  <Icon
+                    src={action.icon}
+                    className="icon-size-24"
+                  />
+                  <span
+                    className="font-weight-normal ml-2"
+                    style={{ fontSize: '18px', lineHeight: '28px' }}
```

Comment:
fontSize: 18 and lineHeight: 28px uses multiple places so create a class for it. e.g font-xl,

Reply:
Added class font-xl